File: backend/src/main/java/com/ll/TeamProject/standard/util/Json.java
package com.ll.TeamProject.standard.util;

import com.fasterxml.jackson.databind.ObjectMapper;

public class Json {
    private static final ObjectMapper om = new ObjectMapper();

    public static String toString(Object obj) {
        try {
            return om.writeValueAsString(obj);
        } catch (Exception e) {
            throw new RuntimeException("JSON 직렬화 실패", e);
        }
    }
}

File: backend/src/main/kotlin/com/ll/TeamProject/standard/util/Json.kt
package com.ll.TeamProject.standard.util

import com.fasterxml.jackson.databind.ObjectMapper

object Json {
    private val objectMapper = ObjectMapper()

    fun toString(obj: Any?): String = runCatching {
        objectMapper.writeValueAsString(obj)
    }.getOrElse { e ->
        throw RuntimeException("JSON 직렬화 실패", e)
    }
}
